pxd.h

```diff
@@ -72,7 +85,8 @@ enum pxd_opcode {
 /** Device identification passed from kernel on initialization */
 struct pxd_dev_id {
 	uint32_t local_minor; 	/**< minor number assigned by kernel */
-	uint32_t pad;
+	uint8_t pad[3];
+	uint8_t fastpath:1, blkmq_device:1, suspend:1, unused:5;
```

Comment:
Where are these new fields used?

Reply:
This is being referenced from px-storage, device_manager.cc